src/main/java/software/amazon/cloudformation/scheduler/CloudWatchScheduler.java

```diff
@@ -81,9 +81,10 @@ public void refreshClient() {
      * @param <ResourceT> resource model state configuration to apply
      * @param <CallbackT> callback context associated with reschedule context
      */
-    public <ResourceT, CallbackT> void rescheduleAfterMinutes(final String functionArn,
-                                                              final int minutesFromNow,
-                                                              final HandlerRequest<ResourceT, CallbackT> handlerRequest) {
+    public <ResourceT, CallbackT,
+        ConfigurationT> void rescheduleAfterMinutes(final String functionArn,
+                                                    final int minutesFromNow,
+                                                    final HandlerRequest<ResourceT, CallbackT, ConfigurationT> handlerRequest) {
```

Comment:
is it even used anywhere? @srujithpoondla03

Reply:
Nope, I think there is some cleanup required in this repo for V1 Code(not just this pull request). We did not totally remove the code related to CW events yet.